frontend/lib/src/components/shared/BaseButton/BaseButton.tsx

```diff
@@ -71,12 +72,13 @@ function BaseButton({
   return (
     <ComponentType
       kind={kind}
-      size={size || BaseButtonSize.MEDIUM}
+      size={size ?? BaseButtonSize.MEDIUM}
       fluidWidth={fluidWidth || false}
       disabled={disabled || false}
       onClick={onClick || (() => {})}
       autoFocus={autoFocus || false}
       data-testid={`baseButton-${kind}`}
+      style={style || {}}
```

Comment:
yep that's a great point! I did this because the button kind would change dynamically here, but I can also add two kinds of new buttons (with a telling name or so). I will update this 👍

Reply:
I added an extra BaseButton kind: `BORDERLESS_ICON_ACTIVE`